test/JDBC/expected/babel_varbinary.out

```diff
@@ -164,6 +164,396 @@ go
 ~~ERROR (Message: division by zero)~~
 
 
+--division between varbinary and int4 datatype, vice-versa
+Select 100 / 0x
```

Comment:
Do we have specific test cases where divisor/dividend are of size 1, 2, 3 and 4?

Reply:
Added specific test for size 3 rest  , were present.